.github/workflows/build.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     with:
       image: ghcr.io/${{ github.repository }}
       container-registry: ghcr.io
-      platforms: linux/amd64
+      platforms: linux/amd64,linux/arm/v7
```

Comment:
Nice, can you also add armv8? This way, people can run it on rpi4 and M1 macs

Reply:
this is fix now. plz mrg